view/src/components/foodbanks.js

```diff
@@ -856,11 +825,11 @@ class Foodbank extends Component {
                             Point of Contact:
                           </Typography>
                           <Typography variant="body2" component="p">
-                            Name: {foodbank.contactName}
+                            Name: {foodbank.contacts[0]["contactName"]}
                             <br />
-                            Phone: {foodbank.contactPhone}
+                            Phone: {foodbank.contacts[0]["contactPhone"]}
                             <br />
-                            Email: {foodbank.contactEmail}
+                            Email: {foodbank.contacts[0]["contactEmail"]}
```

Comment:
same comments as the last PR - are we sure this will be defined? do we only want the first contact?

Reply:
iirc when I was running this for farms, it would display a default/sample contact to show the formatting at first -- if we keep this, maybe we can be a bit more specific in the name/organization of the sample contact to make it clear that it is an example person